keyboards/tokyo60/tokyo60.h

```diff
@@ -1,7 +1,9 @@
-#ifndef TOKYO60_H
-#define TOKYO60_H
+#pragma once
 
 #include "quantum.h"
+#include "via.h"
```

Comment:
This file is automatically included if `VIA_ENABLE = yes`; you shouldn't need it here.

```suggestion
```